aim/sdk/adapters/pytorch_lightning.py

```diff
@@ -80,6 +83,7 @@ def experiment(self) -> Run:
                     system_tracking_interval=self._system_tracking_interval,
                     capture_terminal_logs=self._capture_terminal_logs
                 )
+                self._run.name = self._run_name
```

Comment:
Just realized we need to check the `self._run_name` first. Passing value `None` actually means to leave the name as it was, so no need to assign in that case. Thoughts?

Reply:
Agreed! Good catch! :)